src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -83,6 +84,11 @@ const render = function render(interaction) {
                 $el.attr('placeholder', placeholderText);
             }
             if (_getFormat(interaction) === 'xhtml') {
+                
+                if(!features.isVisible('taoQtiItem/creator/interaction/extendedText/property/xhtmlConstraints') && !features.isVisible('taoQtiItem/creator/interaction/extendedText/property/xhtmlRecommendations')) {
```

Comment:
**question:** Is it needed to use such a long flag? The purpose of the visibility feature was to avoid having to use module URIs.

**suggestion:** Would not it be better to have shorter flags, like:
```suggestion
                if(!features.isVisible('extendedTextInteraction/xhtmlConstraints') && !features.isVisible('extendedTextInteraction/xhtmlRecommendations')) {
```

Reply:
We had a sync conversation on that naming, and decided to move forward with the longer ones as because

 - Values won't be changed often (probably once while instance is configured)
 - Most likely keys will be copy-pasted to the config file
 - The key name gives understanding the extension where it is used in